Demographics and disparities subsection (#460)
greenelab/covid19-review: content/02.introduction.md

Individual outcomes have varied significantly, and reported hospitalization rates have been wide-ranging based on age and location, with various underlying health conditions being disproportionately reported [@doi:10.15585/mmwr.mm6915e3].
Outcomes in patients that are hospitalized have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032; @doi:10.1001/jama.2020.2648; @doi:10.1001/jama.2020.4031].

> Though older populations are generally considered the most vulnerable, pediatric infections are also a concern.

Reviewer: There is not basis to say that - far more deaths in older cases and less in pediatric. You cannot compare them.

Author: I agree -- I put this paragraph in the wrong place.  I feel that pediatric infections and MIS/C need to be mentioned somewhere, but all of the other paragraphs in this subsection are about groups who are at a statistically higher risk, and this group is at a lower risk.  Maybe it would be better to move this paragraph into the subsection about clinical presentation.  Do you think so, @rando2?

Reviewer 2: @dziakj1 that sounds reasonable to me! My understanding is that it's not unusual for pediatric cases to present somewhat differently than adult cases, so probably worth distinguishing! I'll see if I can get a pediatrician to read through it when we finish!

Reviewer 2: Resolving this conversation because this topic is moved to #476

> The prevalence of different clinical features of COVID-19 infection differ in different areas of the world, presumably due to differential reporting and/or different levels of preexisting risk factors [@doi:10.1007/s00592-020-01546-0].

Reviewer 2: Hmm maybe we also want some medical people in here! @esell17 @anskelly @rishirajgoel
Also @yemarshall in case you're interested!

> The prevalence of different clinical features of COVID-19 infection differ in different areas of the world, presumably due to differential reporting and/or different levels of pre-existing risk factors [@doi:10.1007/s00592-020-01546-0].

Reviewer 3: I think "pre-existing" may be more of an insurance term, you could just say "risk factors"

Reviewer: I would write ..... "varying levels of pre-exitisting co-morbidities, such as hypertension, diabetes and lung diseases".

Reviewer 3: also to check at some point: how big effect age structure / age distribution may have ... this is also more complicated b/c depends on other possible age-related differences and also differences in age contact structure / interactions between different age groups.

Author: I hadn't thought about age as a confounder -- I will check the reference you suggested.  Thanks!

> In the context of the United States, persons infected with COVID-19 are more likely to require hospitalization if they are of male sex, older age, and/or of Black/African American background [@doi:10.1377/hlthaff.2020.00598; @doi:10.15585/mmwr.mm6925e1].

Reviewer 3: I would like to think about this more, but for now just noting: The wording here might be changed here to make more rigid what's more confirmed or ~rigid and more flexible what should be more flexible. Here's what I mean. It could start instead with the main risk factors that are put by the CDC (age, co-morbidities / disease conditions that could lead to higher risk -- see whatever is the most up to date). Then separately start with a sentence that also describes other possible patterns (not really risk factors, just patterns) seen in the data "so far" -- I say "patterns" because I'm guessing there's still analysis being done and because it's might be more complicated or different to figure out why a pattern related to ~ethnic background or ~race may be seen versus a pattern related to a co-morbidity (although this is also complicated) or age (again although this is also complicated) & I say "so far" is because these patterns might be partially changed if some of the reasons for the patterns are adjustable (e.g. if there's a lack of access --> increase access) and actually all the patterns might be changed because public health sometimes adds targeting (or improves coverage) to populations identified through data as more vulnerable (or more generally fills in coverage gaps).

Author: Thank you, @yemarshall !  One thing we could do is include a few sentences to say that in theory there may be different ways in which a group could have higher risk for COVID-19 deaths or other severe outcomes.  They could:
1.  be more likely to contact the virus (e.g., essential workers, people living in big cities?)
2.  be more likely to get infected once exposed (e.g., depressed immune system?)
3.  be more likely to get seriously sick once infected (e.g., comorbidities, older age?)
4.  be less likely to get adequate help once seriously sick (e.g., poor, minorities, far from health care).
These can all get lumped together because some of the steps aren't easy to empirically study (e.g., it seems hard to study #2 directly without doing some really horrible human challenge study, and by definition of #4 the people aren't around to be studied by the scientists).  So in practice they add up to the same thing, but in terms of mechanisms they are completely different.  In a few cases the distinction might be especially important (e.g., is a group less resistant to the virus for genetic reasons, or are they living in more crowded places, or are they not getting enough information, or not enough care -- if we knew for sure which one, it would in principle be easier to design an intervention).   Of course we could never resolve this in one subsection in a literature review, but maybe it would be a good contribution to list the questions?  What do you think, @yemarshall and @rando2?

Reviewer 2: @dziakj1 I like that breakdown! Probably outside the scope of this section, but there is also a loop from 4 to 2 due to allostatic load, i.e. the long-term burden of chronic stress -- assuming that reduced to access to health resources is correlated with other life stressors such as experiencing racism or poverty.

Potentially framing it as something along the lines of:
There are some factors that are associated with greater risk from COVID-19. There are several possible etiologies through which a factor could influence COVID-19 outcomes, which could (your list of 4).

(I am not really sure if this is the best way to phrase this because I am not a public health person!)

> In the context of the United States, persons infected with COVID-19 are more likely to require hospitalization if they are of male sex, older age, and/or of Black/African American background [@doi:10.1377/hlthaff.2020.00598; @doi:10.15585/mmwr.mm6925e1].

Reviewer: In the context of the United States, persons infected with COVID-19 are more likely to require hospitalization if they are of male, older age, and/or have pre-existing conditiosn. Black/African American backgrounds have a higher risks than White Americans and are younger[@doi:10.1377/hlthaff.2020.00598; @doi:10.15585/mmwr.mm6925e1].

(Blacks/African American are younger than Whites!!!!)

> In the context of the United States, persons infected with COVID-19 are more likely to require hospitalization if they are of male sex, older age, and/or of Black/African American background [@doi:10.1377/hlthaff.2020.00598]. [@doi:10.15585/mmwr.mm6925e1]

Reviewer 2: ```suggestion
In the context of the United States, persons infected with COVID-19 are more likely to require hospitalization if they are of male sex, older age, and/or of Black/African American background [@doi:10.1377/hlthaff.2020.00598; @doi:10.15585/mmwr.mm6925e1].
```

> African Americans have also been reported to have disproportionate risk for kidney complications for COVID-19. [@doi:10.1016/j.kint.2020.05.006]  

Reviewer 2: ```suggestion
African Americans have also been reported to have disproportionate risk for kidney complications for COVID-19 [@doi:10.1016/j.kint.2020.05.006].
```

> The racial disparity may be due to a number of factors, including different vulnerability due to higher levels of a pre-existing conditions such as diabetes, to greater economic difficulties or more hazardous work, or to less access to or trust in the health care system [@doi:10.1377/hlthaff.2020.00598; @doi:10.1016/j.kint.2020.05.006].

Reviewer 3: putting "trust" in there without a specific description like what's in the article cited doesn't work as well / is too general (and is too rigid b/c it can be changed); maybe something like "different pattern of use (e.g. for just severe conditions or for both mild and severe conditions)"

Reviewer: The racial disparity may be due to a number of factors, including greater economic difficulties or more hazardous work and due to higher levels of pre-existing conditions such as hypertension or diabetes. These groups also have less access to the health care system which may cause this disproportionate higher burden of coronavirus [@doi:10.1377/hlthaff.2020.00598; @doi:10.1016/j.kint.2020.05.006].

> The racial disparity may be due to a number of factors, including different vulnerability due to higher levels of a pre-existing conditions such as diabetes, to greater economic difficulties or more hazardous work, or to less access to or trust in the health care system.[@doi:10.1377/hlthaff.2020.00598; @doi:10.1016/j.kint.2020.05.006]

Reviewer 2: ```suggestion
The racial disparity may be due to a number of factors, including different vulnerability due to higher levels of a pre-existing conditions such as diabetes, to greater economic difficulties or more hazardous work, or to less access to or trust in the health care system [@doi:10.1377/hlthaff.2020.00598; @doi:10.1016/j.kint.2020.05.006].
```

Less health care access could cause infections to be less likely to be diagnosed unless or until they are very severe. 
However, more research is needed into studying and and remediating these disparities. 

> Long-term damage caused by chronic stress related to aversive social experiences [@doi:10.1101/2020.05.10.20097253] leading to cardiovascular risk [@doi:10.1007/s12170-013-0338-5] might be relevant but has not yet been researched in the specific case of COVID-19 disparities.

Reviewer 3: the first reference you cite here should be placed elsewhere in the sentence because it doesn't really discuss what is immediately preceding it

> Disproportionate harm from COVID-19 has also been noted in other minorities, namely Latino/Hispanic communities and Native American / Alaskan Native tribal communities [@doi:10.1001/jama.2020.8598; @doi:10.1136/bmj.m1483; @doi:10.1111/jrh.12451]. 

Author: Yes, it definitely would!  I will do so!

> In particular, some children have experienced a severe inflammatory syndrome after COVID-19 infection, similar in some respects to Kawasaki disease or to Kawasaki disease shock syndrome [@doi:10.1093/jpids/piaa069;@doi:10.1001/jama.2020.10369; @doi:10.1016/j.ajem.2020.05.117].

Reviewer: "A very small proportion of children"..... (kawaski was a rare occurrence)

> Heart failure can also occur [@doi:10.1161/CIRCULATIONAHA.120.048360].

Reviewer: this sentence is a bit out of place.

> Genetic factors may also play a role in risk of respiratory failure for COVID-19 [@doi:10.1056/NEJMoa2020283; @doi:10.1093/gerona/glaa131; @doi:10.1101/2020.06.16.155101], including possibly higher risk to individuals of blood group A and possibly lower risk to individuals of type O [@doi:10.1056/NEJMoa2020283].

Reviewer: The blood group has not been able to replicate in a larger study. I would leave it out.

Diabetes may increase the risk of lengthy hospitalization [@doi:10.1016/j.cmet.2020.04.021] or of death [@doi:10.1016/j.cmet.2020.04.021;@doi:10.1007/s00592-020-01546-0].  
[@doi:10.1111/dom.14057] and [@doi:10.1152/ajpendo.00124.2020] discuss possible ways in which COVID-19 and diabetes may interact.  
On a speculative basis, the mechanisms described in a report by [@doi:10.1007/s00592-009-0109-4] involving effects on the pancreas by the related SARS-CoV-1 virus, which could reportedly cause short-term diabetic symptoms, might perhaps be relevant here as well.

> Obesity also appears to be associated with higher risk of severe outcomes from SARS-CoV-2 [@doi:10.1016/j.metabol.2020.154262; @doi:10.1101/2020.04.23.20076042], although further research is needed regarding the exact mechanisms [@doi:10.1016/j.medj.2020.06.005].

Reviewer: Obesity also appears to be associated with a higher risk of severe outcomes from SARS-CoV-2...... Obesity is considered an underlying risk factor for most of the pre-existing conditions and therefore its causal routes are still under investigation. Further research might be required to know the exact mechanisms.

Because different subpopulations may have somewhat different vulnerabilities, needs, and resources, we recommend that researchers publishing studies on diagnostics and therapeutics take extra care to be clear about the demographic and medical characteristics of their sample, in order to facilitate discussions of the degree to which results may generalize or differ in other settings.

#### Molecular Mechanisms of COVID-19

> - How can we evaluate the human response to SARS-CoV-2 on a molecular level?

Reviewer: There are some studies in cases/controls of hospitalized Covid patients. They can see a cluster of six genes that are mostly related to respiratory inflammation.

Heart failure can also occur [@doi:10.1161/CIRCULATIONAHA.120.048360].

Genetic factors may also play a role in risk of respiratory failure for COVID-19 [@doi:10.1056/NEJMoa2020283; @doi:10.1093/gerona/glaa131; @doi:10.1101/2020.06.16.155101], including possibly higher risk to individuals of blood group A and possibly lower risk to individuals of type O [@doi:10.1056/NEJMoa2020283].

Diabetes may increase the risk of lengthy hospitalization [@doi:10.1016/j.cmet.2020.04.021] or of death [@doi:10.1016/j.cmet.2020.04.021;@doi:10.1007/s00592-020-01546-0].  
[@doi:10.1111/dom.14057] and [@doi:10.1152/ajpendo.00124.2020] discuss possible ways in which COVID-19 and diabetes may interact.  
On a speculative basis, the mechanisms described in a report by [@doi:10.1007/s00592-009-0109-4] involving effects on the pancreas by the related SARS-CoV-1 virus, which could reportedly cause short-term diabetic symptoms, might perhaps be relevant here as well.

> Obesity also appears to be associated with higher risk of severe outcomes from SARS-CoV-2, [@doi:10.1016/j.metabol.2020.154262; @doi:10.1101/2020.04.23.20076042] although 
> further research is needed regarding the exact mechanisms [@doi:10.1016/j.medj.2020.06.005].

Reviewer 2: ```suggestion
Obesity also appears to be associated with higher risk of severe outcomes from SARS-CoV-2, [@doi:10.1016/j.metabol.2020.154262; @doi:10.1101/2020.04.23.20076042] although further research is needed regarding the exact mechanisms [@doi:10.1016/j.medj.2020.06.005].
```

> Because different subpopulations may have somewhat different vulnerabilities, needs, and resources, we recommend that researchers publishing studies on diagnostics and therapeutics take extra care to be clear about the demographic and medical characteristics of their sample, in order to facilitate discussions of the degree to which results may generalize or differ in other settings.

Reviewer 4: That last sentence might be useful for a conclusion section if we have one @rando2

Reviewer 2: Talked to Casey -- we will add a conclusion but probably not for a week or two since anything could change by then!

Individual outcomes have varied significantly, and reported hospitalization rates have been wide-ranging based on age and location, with various underlying health conditions being disproportionately reported [@doi:10.15585/mmwr.mm6915e3].
Outcomes in patients that are hospitalized have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032; @doi:10.1001/jama.2020.2648; @doi:10.1001/jama.2020.4031].

#### Subpopulations of Special Concern  

> In the context of the United States, persons infected with COVID-19 are more likely to require hospitalization if they are of male sex, of older age, and/or of Black/African American background [@doi:10.1377/hlthaff.2020.00598; @doi:10.15585/mmwr.mm6925e1].  African Americans have also  been reported to have disproportionate risk of kidney complications from COVID-19 [@doi:10.1016/j.kint.2020.05.006].  In addition to African Americans, disproportionate harm and mortality from COVID-19 has also been noted in other minorities, namely Latino/Hispanic communities and Native American / Alaskan Native communities such as the Navajo nation [@doi:10.1001/jama.2020.8598; @doi:10.1136/bmj.m1483; @doi:10.1111/jrh.12451; https://www.nytimes.com/2020/04/09/us/coronavirus-navajo-nation.html?searchResultPosition=10; @url:https://www.nytimes.com/interactive/2020/07/05/us/coronavirus-latinos-african-americans-cdc-data.html; @doi:10.1093/tbm/ibaa055]. In Brazil, where the pandemic has also been severe, indigenous communities are likewise at special risk [@doi:10.1126/science.abc0073].

Reviewer 3: maybe double-check the wording of this first sentence (go back to references, including the 14 states study referenced in the second reference you put) and make sure you really mean "... persons infected with COVID-19 are more likely to require hospitalization if they are..." [check if this is what u meant & if it accounts for possible differences in detecting SARS-CoV-2 infection] rather than something like
"... persons hospitalized for COVID-19 are more likely..."

Author: Thanks!  I am checking, and I think that you are right.  I should have said "persons diagnosed" rather than "persons infected" because an undiagnosed infection obviously would never make it into a research study, so we don't know anything about it.  I very much suspect that the outcome disparities also exist for infected people in general, not just diagnosed, but almost by definition there's no data that could prove it.  I will correct the wording.

> The sizable racial disparities observed may be due to a number of factors, including different   levels of pre-existing co-morbidities, such as hypertension, diabetes or lung diseases. They may also be due in part to greater economic difficulties or more hazardous or crowded work or living conditions, or poorer access to health care [@doi:10.1377/hlthaff.2020.00598; @doi:10.1016/j.kint.2020.05.006; @doi:10.1001/jamanetworkopen.2020.12403]. This might cause infections to be less likely to be diagnosed unless or until they were very severe; in the the sample studied by [@doi: 10.1377/hlthaff.2020.00598], African Americans were more likely to be diagnosed in hospital, while other groups were more likely to have been diagnosed in ambulatory settings in the community. Long-term damage caused by chronic stress related to traumatic social experiences [@doi:10.1007/s12170-013-0338-5], perhaps mediated by cardiovascular risk factors, [@doi:10.1101/2020.05.10.20097253], might be relevant but has not yet been researched in the specific case of COVID-19 disparities.  More research is needed in studying and remediating these disparities [@doi: 10.1093/tbm/ibaa055]. 

Author: Sorry, I missed the spacing again!

@doi:10.1371/journal.pone.0232452; @url:www.ons.gov.uk/peoplepopulationandcommunity/healthandsocialcare/causesofdeath/bulletins/coronaviruscovid19relateddeathsbyoccupationenglandandwales/deathsregistereduptoandincluding20april2020]

Many other factors influence risk of serious COVID-19 outcomes.  Genetic factors may play a role in risk of respiratory failure for COVID-19 [@doi:10.1056/NEJMoa2020283; @doi:10.1093/gerona/glaa131; @doi:10.1101/2020.06.16.155101].  Diabetes may increase the risk of lengthy hospitalization [@doi:10.1016/j.cmet.2020.04.021] or of death [@doi:10.1016/j.cmet.2020.04.021;@doi:10.1007/s00592-020-01546-0].  [@doi:10.1111/dom.14057] and [@doi:10.1152/ajpendo.00124.2020] discuss possible ways in which COVID-19 and diabetes may interact.  Obesity also appears to be associated with higher risk of severe outcomes from SARS-CoV-2 [@doi:10.1016/j.metabol.2020.154262; @doi:10.1101/2020.04.23.20076042]. Obesity is considered an underlying risk factor for other health problems, and the mechanism for its contributions to COVID-19 hospitalization or mortality is not yet clear [@doi:10.1016/j.medj.2020.06.005].

> Study of disparities and differences among groups in rates of hospitalization or death from COVID-19 are complicated by the fact that risk may be elevated in multiple ways.  An individual may be more likely to contact the virus, more likely to get infected once exposed, be more likely to have serious complications once infected, or be less likely to get adequate help once they are seriously ill. Although these are difficult to disaggregate, they suggest different possible interventions and each deserves consideration.

Reviewer 3: typo: "An individual may be more likely to": "contract" instead of "contact"

Author: Thanks!  I think that by "contact" I really meant "contact" (be exposed to), but this turns out to be a terrible choice of words because it looks like a typo for "contract."  So I changed "contact" to "be exposed to."

The epidemiological characteristics and clinical presentation of COVID-19 may differ somewhat in different areas of the world, presumably due to differential reporting, different age structures, and/or different risk factors [@doi:10.1007/s00592-020-01546-0].  Furthermore, because different subpopulations may have somewhat different vulnerabilities, needs, and resources, we recommend that researchers publishing studies on diagnostics and therapeutics take extra care to be clear about the demographic and medical characteristics of their sample, in order to facilitate discussions of the degree to which results may generalize or differ in other settings.

Besides the direct harms caused by infection, many populations are in indirect risk of serious harm due to the social and economic effects of the pandemic and of the efforts needed to fight it.  These might include 

> individuals with substance use disorder [@doi:10.7326/M20-1141], victims of natural disasters [@doi: 10.1038/s41558-020-0804-2], and victims of human trafficking [@doi: 10.1016/j.eclinm.2020.100409].  The pandemic could also delay the fight against other major infectious diseases, such as HIV, malaria and tuberculosis, potentially leading to a further increase of mortality [@doi:10.25561/78670]. Although they are beyond the scope of this paper, further research is needed in order to prevent these harms.

Reviewer 3: Format: There are 3 sentences here in 1 line 174. Separate into 3 lines.

Author: Thanks to @rando2 for fixing the spacing!  If even my sentences can't socially distance, then I am indeed in peril... :-)

Individual outcomes have varied significantly, and reported hospitalization rates have been wide-ranging based on age and location, with various underlying health conditions being disproportionately reported [@doi:10.15585/mmwr.mm6915e3].
Outcomes in patients that are hospitalized have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032; @doi:10.1001/jama.2020.2648; @doi:10.1001/jama.2020.4031].

#### Subpopulations of Special Concern  

> In the context of the United States, persons infected with COVID-19 are more likely to require hospitalization if they are of male sex, of older age, and/or of Black/African American background [@doi:10.1377/hlthaff.2020.00598; @doi:10.15585/mmwr.mm6925e1].

Reviewer 2: ```suggestion
A number of interconnected factors appear to influence population-level trends in the impact and severity of SARS-CoV-2.
In the context of the United States, persons infected with COVID-19 are more likely to require hospitalization if they are of male sex, of older age, and/or of Black/African American background [@doi:10.1377/hlthaff.2020.00598; @doi:10.15585/mmwr.mm6925e1].
```

Author: Hello @rando2 !  I was thinking about the term "minoritized."  I have heard it before, but it sounds very awkward as a neologism to me -- I assume it is supposed to mean "marginalized," but I'm not sure why people don't use that word instead if that's what they mean.  I thought of "minority" as being a statistical word, just meaning a group is smaller in number than some other group.  But in that case, I don't know of any data for minoritized groups in the United States OTHER than African American, Hispanic or Native American.  Whether, say, Muslim Jewish or LGBT or autistic people are at higher risk for COVID because some people mistreat them, is an empirical question which as far as I know nobody asked.  It might be true but we don't have any data.  So I think it might make more sense to just say "Hispanic/Latino and Native American" and just drop "other minoritized populations such as."  Would that be okay?

Author: In other words, since we only know about those three groups, maybe they should be the only ones we mention.

Author: I did just recently read about a similar but far larger study done in the UK:
https://doi.org/10.1038/s41586-020-2521-4
They were actually able to estimate P(death) instead of P(death | diagnosis) because they used a massive database of millions of medical records.  The findings were similar but sturdier than the papers we cite here, because of the larger and less selected sample.  So I will add a sentence about it if it is okay with you.  Sorry to keep adding references!  I will add it soon so that you can do the merging tonight if you want.

African Americans have also  been reported to have disproportionate risk of kidney complications from COVID-19 [@doi:10.1016/j.kint.2020.05.006].
In addition to African Americans, disproportionate harm and mortality from COVID-19 has also been noted in other minorities, namely Latino/Hispanic communities and Native American / Alaskan Native communities such as the Navajo nation [@doi:10.1001/jama.2020.8598; @doi:10.1136/bmj.m1483; @doi:10.1111/jrh.12451; https://www.nytimes.com/2020/04/09/us/coronavirus-navajo-nation.html?searchResultPosition=10; @url:https://www.nytimes.com/interactive/2020/07/05/us/coronavirus-latinos-african-americans-cdc-data.html; @doi:10.1093/tbm/ibaa055].
In Brazil, where the pandemic has also been severe, indigenous communities are likewise at special risk [@doi:10.1126/science.abc0073].

> The sizable racial disparities observed may be due to a number of factors, including different levels of pre-existing co-morbidities, such as hypertension, diabetes or lung diseases.

Author: Well, I would defer to your friend, but the chronic stress explanation was supposed to be a mechanism for how being a minority hurts you, so I am not sure it would help to move it.

> The sizable racial disparities observed may be due to a number of factors, including different levels of pre-existing co-morbidities, such as hypertension, diabetes or lung diseases.

Reviewer 2: @tlukan is this the line your previous comment was referring to?

> They may also be due in part to greater economic difficulties or more hazardous or crowded work or living conditions, or poorer access to health care [@doi:10.1377/hlthaff.2020.00598; @doi:10.1016/j.kint.2020.05.006; @doi:10.1001/jamanetworkopen.2020.12403].

Author: Yes, that change is okay.

This might cause infections to be less likely to be diagnosed unless or until they were very severe; in the the sample studied by [@doi: 10.1377/hlthaff.2020.00598], African Americans were more likely to be diagnosed in hospital, while other groups were more likely to have been diagnosed in ambulatory settings in the community.

> Long-term damage caused by chronic stress related to traumatic social experiences [@doi:10.1007/s12170-013-0338-5], perhaps mediated by cardiovascular risk factors, [@doi:10.1101/2020.05.10.20097253], might be relevant but has not yet been researched in the specific case of COVID-19 disparities.

Reviewer 3: I also had not previously seen research like this; if you do use it, I would suggest verifying whether the articles pointed out the more commonly mentioned factors for cardiovascular diseases (which could also be checked to see if they are associated with socio-economic factors or race) like diet, exercise, and smoking. This is just what I hear more commonly discussed, so I'm not saying to write this, I'm just saying to check. E.g. for diet: people for a while have been working to bring more healthy food throughout the US & also there's the possible factor of how much time you have to cook if you work long hours; similarly, for exercise there's the factor of how much time you have for exercise if you work long hours; & smoking may be related to stress?? (just a guess) with smoking being a more commonly mentioned factor for cardiovascular disease.  Also, more commonly I hear that stress can exacerbate chronic physical conditions but I hear less commonly that stress can cause chronic physical conditions.  Again, this is not based on a statistical assessment of research; this is based on what I commonly hear.

> Long-term damage caused by chronic stress related to traumatic social experiences [@doi:10.1007/s12170-013-0338-5], perhaps mediated by cardiovascular risk factors, [@doi:10.1101/2020.05.10.20097253], might be relevant but has not yet been researched in the specific case of COVID-19 disparities.

Reviewer 2: This is based on allostatic load and the biology of stress and trauma. One really influential scientist in this area is Clarence Gravlee (e.g., https://onlinelibrary.wiley.com/doi/10.1002/ajpa.20983)
People don't talk about it enough!

> More research is needed in studying and remediating these disparities [@doi: 10.1093/tbm/ibaa055]. 

Author: Maybe "analyzing"?  But "identifying" is okay.

> People living in certain locations may be especially vulnerable to harm.

Author: I guess that makes sense.  They aren't really separate (in Navajo country there are a lot of Navajos; and in poor neighborhoods there are a lot of poor people), but at least some studies were able to distinguish between them, at least as a community-level covariate.

> People living in certain locations may be especially vulnerable to harm.

Reviewer 2: @dziakj1 Ah! That's what I was trying to ask about but it seems like you already thought about it! Would it make sense to keep your original sentence but just combine with the previous paragraph?

In a preprint, [@doi:10.1101/2020.04.05.20054502] provided observational evidence that geographical areas in the United States which suffer from worse air pollution by fine particulate matter have also suffered more COVID-19 deaths per capita, after adjusting for demographic covariates.

> Although lack of individual-level exposure data and the impossibility of randomization make it difficult to elucidate the exact causal mechanism, this finding would be consistent with similar findings for all-cause mortality (e.g., [@doi:10.1073/pnas.1803222115]).  Individuals in nursing homes / skilled nursing facilities [@doi:10.1001/jama.2020.11642] and in some prisons and detention centers [@doi:10.1001/jamainternmed.2020.1856; @doi:10.1001/jama.2020.12528] have also been exposed to higher risk of infection.

Reviewer 2: ```suggestion
Although lack of individual-level exposure data and the impossibility of randomization make it difficult to elucidate the exact causal mechanism, this finding would be consistent with similar findings for all-cause mortality (e.g., [@doi:10.1073/pnas.1803222115]).
Individuals in nursing homes / skilled nursing facilities [@doi:10.1001/jama.2020.11642] and in some prisons and detention centers [@doi:10.1001/jamainternmed.2020.1856; @doi:10.1001/jama.2020.12528] have also been exposed to higher risk of infection.
```

> Certain occupations, such as health care work, can put individuals at increased risk [@doi:10.1002/ajim.23145; 
> @doi:10.1371/journal.pone.0232452; @url:www.ons.gov.uk/peoplepopulationandcommunity/healthandsocialcare/causesofdeath/bulletins/coronaviruscovid19relateddeathsbyoccupationenglandandwales/deathsregistereduptoandincluding20april2020]

Reviewer 2: ```suggestion
Certain occupations, such as health care work, can put individuals at increased risk [@doi:10.1002/ajim.23145; @doi:10.1371/journal.pone.0232452; @url:www.ons.gov.uk/peoplepopulationandcommunity/healthandsocialcare/causesofdeath/bulletins/coronaviruscovid19relateddeathsbyoccupationenglandandwales/deathsregistereduptoandincluding20april2020].
```

Many other factors influence risk of serious COVID-19 outcomes. 
Genetic factors may play a role in risk of respiratory failure for COVID-19 [@doi:10.1056/NEJMoa2020283; @doi:10.1093/gerona/glaa131; @doi:10.1101/2020.06.16.155101].
Diabetes may increase the risk of lengthy hospitalization [@doi:10.1016/j.cmet.2020.04.021] or of death [@doi:10.1016/j.cmet.2020.04.021;@doi:10.1007/s00592-020-01546-0].
[@doi:10.1111/dom.14057] and [@doi:10.1152/ajpendo.00124.2020] discuss possible ways in which COVID-19 and diabetes may interact.

> Obesity also appears to be associated with higher risk of severe outcomes from SARS-CoV-2 [@doi:10.1016/j.metabol.2020.154262; @doi:10.1101/2020.04.23.20076042]. Obesity is considered an underlying risk factor for other health problems, and the mechanism for its contributions to COVID-19 hospitalization or mortality is not yet clear [@doi:10.1016/j.medj.2020.06.005].

Reviewer 2: ```suggestion
Obesity also appears to be associated with higher risk of severe outcomes from SARS-CoV-2 [@doi:10.1016/j.metabol.2020.154262; @doi:10.1101/2020.04.23.20076042].
Obesity is considered an underlying risk factor for other health problems, and the mechanism for its contributions to COVID-19 hospitalization or mortality is not yet clear [@doi:10.1016/j.medj.2020.06.005].
```

In addition to African Americans, disproportionate harm and mortality from COVID-19 has also been noted in Latino/Hispanic communities and in Native American / Alaskan Native communities such as the Navajo nation [@doi:10.1001/jama.2020.8598; @doi:10.1136/bmj.m1483; @doi:10.1111/jrh.12451; https://www.nytimes.com/2020/04/09/us/coronavirus-navajo-nation.html?searchResultPosition=10; @url:https://www.nytimes.com/interactive/2020/07/05/us/coronavirus-latinos-african-americans-cdc-data.html; @doi:10.1093/tbm/ibaa055].
In Brazil, where the pandemic has also been severe, indigenous communities are likewise at special risk [@doi:10.1126/science.abc0073].
The sizable racial disparities observed may be due to a number of factors, including different levels of pre-existing co-morbidities, such as hypertension, diabetes or lung diseases.
They may also be influenced by the disproportionate socioeconomic burdens placed on many people of color, which can correspond to greater economic difficulties, more hazardous or crowded work or living conditions, or reduced access to health care [@doi:10.1377/hlthaff.2020.00598; @doi:10.1016/j.kint.2020.05.006; @doi:10.1001/jamanetworkopen.2020.12403].

> This might cause infections to be less likely to be diagnosed unless or until they were very severe; in the the sample studied by [@doi: 10.1377/hlthaff.2020.00598], African Americans were more likely to be diagnosed in hospital, while other groups were more likely to have been diagnosed in ambulatory settings in the community.

Reviewer 2: ```suggestion
This might cause infections to be less likely to be diagnosed unless or until they were very severe; in the the sample studied by [@doi:10.1377/hlthaff.2020.00598], African Americans were more likely to be diagnosed in hospital, while other groups were more likely to have been diagnosed in ambulatory settings in the community.
```

This was done using a dataset of 17 million adults in the United Kingdom.  
In this administrative dataset, [@doi:10.1038/s41586-020-2521-4] found that male sex, older age, economic deprivation, nonwhite ethnicity (principally Black or South Asian), obesity, diabetes,  asthma, and several other comorbid conditions were associated with a higher risk of death from COVID-19. 
They cautioned that despite the large sample size, their observational data did not directly provide information on the causal mechanisms which led to these statistical associations.  
However, the ethnic and economic disparities did not vanish when adjusting for observed preexisting conditions.  
[@doi:10.1038/s41586-020-2521-4] also found that the predictive value of socioeconomic status seemed to increase over time, perhaps because many higher-paid workers were able to transition to working online.
 
Besides the direct harms caused by infection, many populations are in indirect risk of serious harm due to the social and economic effects of the pandemic and of the efforts needed to fight it.  

> These might include individuals with substance use disorder [@doi:10.7326/M20-1141], victims of natural disasters [@doi: 10.1038/s41558-020-0804-2], and victims of human trafficking [@doi: 10.1016/j.eclinm.2020.100409].

Reviewer 2: ```suggestion
These might include individuals with substance use disorder [@doi:10.7326/M20-1141], victims of natural disasters [@doi:10.1038/s41558-020-0804-2], and victims of human trafficking [@doi:10.1016/j.eclinm.2020.100409].
```

Individual outcomes have varied significantly, and reported hospitalization rates have been wide-ranging based on age and location, with various underlying health conditions being disproportionately reported [@doi:10.15585/mmwr.mm6915e3].
Outcomes in patients that are hospitalized have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032; @doi:10.1001/jama.2020.2648; @doi:10.1001/jama.2020.4031].

#### Subpopulations of Special Concern  

> In the context of the United States, persons diagnosed with COVID-19 are more likely to require hospitalization if they are of male sex, of older age, and/or of Black/African American background [@doi:10.1377/hlthaff.2020.00598; @doi:10.15585/mmwr.mm6925e1].

Reviewer 3: I don't see our earlier comments/conversation about this. When I browsed the articles you cited & the 14 state CDC article cited in the second article, then I also saw it written something like:
"Hospitalized COVID-19 patients are more commonly older, male, of black race, and have underlying conditions."
or similarly
"Reports of U.S. patients hospitalized with SARS-CoV-2 infection (the virus that causes COVID-19) describe high proportions of older, male, and black persons (2-4)." followed by another sentence about underlying conditions
[quotations from summary & intro from second reference]
I think it makes sense from the beginning to cover both types of major observations seen so far -- things having to do with probability of getting infected and things having to do with probability that infection is severe. You do this later on, but I think your opening sentence should also include both in order to get readers to see both key interesting points from the beginning. If you want just one sentence then I think something closer to the quotation above is better because it covers both aspects and then it can be broken up into components later on; the other option is putting both your current sentence & something like quotation above to suggest multiple components from beginning. (Though everyone has worked on this, still going back to this discussion b/c it's at beginning of section)

Author: Thank you, @yemarshall!  I guess the ideal thing to do would be to tell which demographic, lifestyle, or medical variables contribute to each stage of the process, like:
1. Risk factors for infection:...
2. Risk factors for hospitalization given infection:...
3. Risk factors for death given hospitalization:...
Then we could talk about, for example, are the elderly more likely to get infected than the young, or do they just get sicker if and when they do get infected?  
But I'm not sure whether we can really do that with the literature we have now, which is preliminary and doesn't have prospective longitudinal data on the general population.  Studies on hospitalized samples can provide information about #3 but only #3.  Studies using vital statistics can probably provide information on a muddled mixture of 1, 2, and 3, but probably only under assumptions that aren't safe to make (equal and honest reporting by the authorities in all localities). 
We certainly could talk about these general issues first, and then mention the specific disparities and risk factors.  But it might be kind of embarrassing for us in a way, because we will have to say "Here are some important questions!  And the answers are ... we don't know."  So I thought of putting the theoretical issues at the end of the subsection instead, like the limitations section at the end of a paper.  I guess that doesn't really solve the problem either -- we can only say "Here's a bunch of findings!  By the way, it would be nice if we could explain them."  @rando2, what do you think?

Reviewer 2: @dziakj1 would these perhaps be questions worth raising in the discussion? #514 
This is a huge area of research and definitely we are not going to be able to answer these questions. I made the discussion file specifically to support brainstorming on this topic because it seems like there are far more questions than answers and a lot of these concerns may be easier to address as outstanding concerns in the discussion!

Author: You're right, @rando2.  I know that the deadline is looming ever closer!  Would you like to just reorganize this section however you see fit?  I will approve whatever changes you make.

Reviewer 2: Hi @dziakj1 I'm happy to! I'm a lot sharper in the morning so I'll work on this first thing tomorrow.

The epidemiological characteristics and clinical presentation of COVID-19 may differ somewhat in different areas of the world, presumably due to differential reporting, different age structures, and/or different risk factors [@doi:10.1007/s00592-020-01546-0].
Furthermore, because different subpopulations may have somewhat different vulnerabilities, needs, and resources, we recommend that researchers publishing studies on diagnostics and therapeutics take extra care to be clear about the demographic and medical characteristics of their sample, in order to facilitate discussions of the degree to which results may generalize or differ in other settings.

Several studies on disparities in COVID-19 have compared groups in terms of their probability of a severe outcome (like hospitalization or death) given that a person is diagnosed.  
The larger population of all exposed or all infected people usually cannot directly be studied, or even identified, because of the existence of undetected infections.  

> In contrast, [@doi:10.1038/s41586-020-2521-4] were a le to compare overall rates of death from COVID-19, not simply in the subsample of diagnosed individuals, thus combining the probability of infection with the probability of death given infection.  

Reviewer 2: ```suggestion
In contrast, [@doi:10.1038/s41586-020-2521-4] were able to compare overall rates of death from COVID-19, not simply in the subsample of diagnosed individuals, thus combining the probability of infection with the probability of death given infection.  
```

This was done using a dataset of 17 million adults in the United Kingdom.  

> In this administrative dataset, [@doi:10.1038/s41586-020-2521-4]  found that male sex, older age, economic deprivation, nonwhite ethnicity (principally Black or South Asian), obesity, diabetes,  asthma, and several other comorbid conditions were associated with a higher risk of death from COVID-19.  

Reviewer 2: ```suggestion
In this administrative dataset, [@doi:10.1038/s41586-020-2521-4] found that male sex, older age, economic deprivation, nonwhite ethnicity (principally Black or South Asian), obesity, diabetes,  asthma, and several other comorbid conditions were associated with a higher risk of death from COVID-19. 
```

They cautioned that despite the large sample size, their observational data did not directly provide information on the causal mechanisms which led to these statistical associations.  
However, the ethnic and economic disparities did not vanish when adjusting for observed preexisting conditions.  

> They also found some evidence that the risk associated with socioeconomic evidence may have increased with time, perhaps because many workers in jobs which cannot be done online also have lower-paying jobs.

Reviewer 2: "evidence" doesn't seem like quite the right word -- variables, maybe? I'm not quite sure what would fit here.

Author: I totally botched that sentence.  How about "They found that the predictive value of socioeconomic status seemed to increase over time, perhaps because many higher-paid workers were able to transition to working online."

> 

Reviewer 2: ```suggestion
```
